ospology-live/2025-march-amsterdam/README.md

```diff
@@ -9,8 +9,11 @@ Links to slides will be added to this list once speakers submit them through GH
 - [From Policy to Action: The challenges and successes of Open Source within the Dutch government - Boris van Hoytema and Koos Steenbergen, Dutch Interior Ministry](https://github.com/todogroup/ospology/discussions/564#discussioncomment-12625396)
 - [Measuring the sustainability and maintainability of the dependencies in your software projects - Ana Jimenez Santamaria, TODO Group and Miguel Ángel Fernández, Bitergia](https://github.com/todogroup/ospology/discussions/564#discussioncomment-12631895)
 - [Open Source in Research & Education - Corno Vromans, SURF](https://github.com/todogroup/ospology/discussions/564#discussioncomment-12631984)
+- [OpenSSF Security Baseline](https://docs.google.com/presentation/d/10jDMYmWDIfo5HAUA8lonkJRbQiszx28_-esEYNAGCIM/edit?slide=id.p#slide=id.p)
```

Comment:
This presentation is private

Reply:
@sgoggins could you ask Daniel to update view permissions of the presentation deck? :)